draft-ietf-rats-corim.md

```diff
@@ -1917,37 +1972,37 @@ Corroboration is the process of determining whether actual Attester state (as co
 
-### Processing Endorsements
-
-Endorsers publish Endorsements using the endorsed values triple ({{sec-comid-triple-endval}}) which are transformed ({{sec-end-trans}}) into an internal representation ({{sec-ir-end-val}}).
+Endorsers publish Endorsements using endorsement triples (see {{sec-comid-triple-endval}, {{sec-comid-triple-cond-endors}}, and {{sec-comid-triple-cond-series}}) which are transformed ({{sec-end-trans}}) into an internal representation ({{sec-ir-end-val}}).
 Endorsements describe actual Attester state.
 Endorsements are added to the ACS if the Endorsement condition is satisifed by the ACS.
 
-Endorsed Values are matched with ACS entries by iterating through the `ev` list.
-For each `ev` entry, the `condition` ECT is compared with an ACS ECT.
-If the ECTs match, the `ev` `addition` ECT is added to the ACS.
-
-### Processing Conditional Endorsements
-
-> [Ned] *This section should be identical to the previous section since Endorsement triples and Conditional Endorsement triples are transformed into the same internal representation based on `ev`*
+### Processing Endorsements {#sec-process-end}
 
-### Processing Conditional Endorsement Series
+Endorsements are matched with ACS entries by iterating through the `ev` list.
+For each `ev` entry, the `condition` ECT is compared with an ACS ECT, where the ACS ECT `cmtype` contains either `evidence` or `endorsements`.
+If the ECTs match ({{sec-match-condition-ect}}), the `ev` `addition` ECT is added to the ACS.
 
-> [Ned] *This section should describe augmentation in the context of the `evs` internal representation*
+### Processing Conditional Endorsements {#sec-process-cond-end}
 
-For each Conditional Endorsement Series Triple the Verifier iterates over the `conditional-series-record`s within the triple, stopping if it finds a match.
+Conditional Endorsement Triples are transformed into an internal representation based on `ev`.
+Conditional endorsements have the same processing steps as shown in ({{sec-process-end}}).
 
-For each iteration, the Verifier creates a temporary `stateful-environment-record` by merging the `stateful-environment-record` in the triple with the `refv` field in the `conditional-series-record`. It compares this temporary record against the ACS (see {{sec-match-condition-ect}}).
+### Processing Conditional Endorsement Series {#sec-process-series}
 
-If one of the temporary records matches then the Verifier MUST add the `endv` Endorsement entry to the ACS.
-This Endorsement includes the authority which signed the Conditional Endorsement Series Triple.
+Conditional Endorsement Series Triples are transformed into an internal representation based on `evs`.
+Conditional series endorsements are matched with ACS entries first by iterating through the `evs` list,
+where for each `ev` entry, the `condition` ECT is compared with an ACS ECT, where the ACS ECT `cmtype` contains either `evidence` or `endorsements`.
```

Comment:
Iterating through the entries with `cmtype` is `endorsement`? There's no relation entries of the ACS, and there's no ev in an evs. This sentence confuses me.

Reply:
It looks like a typo.